packages/fxa-event-broker/.gitignore

```diff
@@ -1,3 +1,21 @@
-config/local.json
-target
-**/*.rs.bk
+/coverage.html
+/node_modules
+/sandbox
+/config/public-key.json
+/config/secret-key.json
+/config/vapid-keys.json
+*.swp
+server.log*
+secret*
+*.gpg
+*.log
+/test/load/bin
+/test/load/lib*
+/test/load/include
+*~
+*.pyc
+.nyc_output
+npm-debug.log
+server.pot
+.mail_output
+fxa-content-server-l10n/
```

Comment:
nit: hm, not sure we need all this here?

Reply:
I definitely copied it from one of the other projects.